www/nginx/src/opnsense/mvc/app/library/OPNsense/Nginx/StreamAccessLogParser.php

```diff
@@ -31,17 +31,28 @@
 class StreamAccessLogParser
 {
     private $file_name;
-    private $lines;
     private $result;
 
     private const LogLineRegex = '/(\S+) \[([\d\sa-z\:\-\/\+]+)\] (\S+?) (\d+) (\d+) (\d+) (\d+(?:\.\d+)?)/i';
 
+
     function __construct($file_name)
     {
         $this->file_name = $file_name;
-        $this->lines = file($this->file_name);
-        $this->result = array_map([$this, 'parse_line'], $this->lines);
+        $this->result = array();
+        $this->parse_file();
     }
+
+    private function parse_file() {
```

Comment:
looks like this would go against style, style-fix should address this

Reply:
the underscore?